tests/unit/util/test_sequence.py

```diff
@@ -67,31 +65,6 @@ def test_get_covid_genome_metadata_url(s3_setup, test_file_path, metadata_file):
     assert isinstance(metadata, pl.LazyFrame)
 
 
-@pytest.mark.parametrize(
```

Comment:
Removed these tests because the function is gone and was been replaced by something that doesn't have any logic (it just downloads data using the requests library)